app-misc/sl/sl-5.02.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="sophisticated graphical program which corrects your miss typing"
+HOMEPAGE="http://www.tkl.iis.u-tokyo.ac.jp/~toyoda/index_e.html https://github.com/mtoyoda/sl/"
+SRC_URI="https://github.com/mtoyoda/${PN}/archive/${PV}.tar.gz"
```

Comment:
You need to use the `-> new-filename.tar.gz` syntax to rename the file. Otherwise it's fetched as `5.02.tar.gz` (as you can see in the Manifest change above).